builtins/parse_config.go

```diff
@@ -134,16 +124,48 @@ func parseCombinedConfigFiles(bctx rego.BuiltinContext, op1 *ast.Term) (*ast.Ter
 	return toAST(bctx, combined["Combined"], content)
 }
 
-func decodeTypedArgs[T any](ty T, args ...*ast.Term) ([]T, error) {
+func decodeSliceArg[T any](arg *ast.Term) ([]T, error) {
+	iface, err := ast.ValueToInterface(arg.Value, nil)
+	if err != nil {
+		return nil, fmt.Errorf("decode arg: %w", err)
+	}
+	ifaceSlice, ok := iface.([]any)
+	if !ok {
+		return nil, fmt.Errorf("decodeSliceArg used with non-slice value: (%T)%v", iface, iface)
```

Comment:
Having `iface` in the return here doesn't seem right, was this intentional?

Reply:
It is showing the type and value of the argument that was given that is not a slice. I thought it could be useful to surface this for quicker understanding of why a call to parse_combined_configs failed.

Reply:
ah right, I missed the %T. I was thinking %v %v